frontend/src/components/common/header/header.module.css

```diff
@@ -0,0 +1,87 @@
+.container {
+  background-color: var(--color-header-background);
+  display: flex;
+  align-items: center;
+  width: 1900px;
+  height: 104px;
+  background-color: var(--color-header-background);
+  color: #fff;
+  text-align: left;
+}
+
+.logoContainer {
+  display: flex;
+  align-items: center;
+  font-weight: 500;
+  font-size: 15.6024px;
+  line-height: 18px;
+  letter-spacing: 0.01em;
+  color: #aeaeae;
+  margin-left: 188px;
+}
+.logoContainerLogo {
+  margin-right: 4px;
+  width: 22px;
+  height: 22px;
+  background-color: #aeaeae;
+  border-radius: 50%;
+}
+
+.navigation {
+  display: flex;
+  height: 21px;
+  justify-content: space-between;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 18px;
+  line-height: 21px;
+  margin-left: 170px;
+}
+.liNavigation {
+  margin-right: 40px;
+  cursor: pointer;
+  list-style: none
+}
+
+.activeLiNavigation {
+  color: var(--color-link-active);
+}
+.activeLiNavigation::after {
+  content: '';
+  display: block;
+  margin-top: 6px;
+  background-color: var(--color-link-active);
+  width: 100%;
+  height: 2px;
+  cursor: auto;
+}
+
+.userInfo {
+  display: flex;
+  align-items: center;
+  margin-left: 742px;
+}
+
+.profile {
+  width: 40px;
+  height: 40px;
+  margin-left: 84px;
+  cursor: pointer;
+}
+.profileAvatar {
+  max-width: 100%;
+  max-height: 100%;
```

Comment:
img {
    max-width: 100%;
    height: auto;
}
please, add it to global style

```suggestion
```